src/sc_kpm/sc_server.py

```diff
@@ -54,7 +58,7 @@ def stop(self) -> None:
 class ScServer(ScServerAbstract):
     def __init__(self, sc_server_url: str) -> None:
         self._url: str = sc_server_url
-        self._modules: list[ScModuleAbstract] = []
+        self._modules: set[ScModuleAbstract] = set()
```

Comment:
In _src/sc_kpm/sc_module.py_ you use import and **Set** type, but here you use casual **set** type. I think better to unify it

Reply:
In this file we use `__future__.annotations`